cmd/backup/script.go

```diff
@@ -218,6 +220,18 @@ func newScript() (*script, error) {
 		s.storages = append(s.storages, azureBackend)
 	}
 
+	if s.c.DropboxToken != "" {
+		dropboxConfig := dropbox.Config{
+			Token:      s.c.DropboxToken,
+			RemotePath: s.c.DropboxRemotePath,
+		}
+		dropboxBackend, err := dropbox.NewStorageBackend(dropboxConfig, logFunc)
+		if err != nil {
+			return nil, err
```

Comment:
Nit: all other errors are wrapped (see below for an example) so we can get a poor man's stack trace.

Reply:
I was just thinking

```go
return nil, fmt.Errorf("newScript: error creating dropbox storage backend: %w", err)
```